### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

File: Navigation.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+body {
+    background-color: aliceblue;
```
Comment (ashishkatiyar2404) [inline Navigation.css]: use variables for colors

File: Navigation.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  }
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    height: 100vh;
+  }
+  .alert-container {
+    align-items: center;
+    background-color: #f0f0f0;
```
Comment (ashishkatiyar2404) [inline Navigation.css]: same here ... use variables

File: Navigation.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  
+  .navbar {
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    border-radius: 5px;
+  }
+  /* .navbar.navbar:second-child {
+    justify-self: center;
+  } */
```
Comment (ashishkatiyar2404) [inline Navigation.css]: you can remove comments

File: Navigation.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  .dropdown {
+    position: relative;
+    display: inline-block;
+  }
+  /* Dropdown Content (Hidden by Default) */
+  .dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #f1f1f1;
+    min-width: 160px;
```
Comment (ashishkatiyar2404) [inline Navigation.css]: use other units if >10px.

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

**Comment by ashishkatiyar2404** [comment]
code is good. easy to readable requested some change pls do. then it will ready to merge💯

File: Navigation.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    z-index: 1;
+    right: -500%;
```
Comment (deekshithmd) [inline Navigation.css:55]: is it required this value to be -500%?

**Comment by j24m** [comment]
Hello, just like other people said, you can use color variables in place of direct colors, and comments can be removed, btw code is really clean and readable so liked it:) Rest is good to merge.

File: Navigation.css (hunk 1 above)
Comment (ShrutiLakhani) [reply Navigation.css]: Done.

File: Navigation.css (hunk 2 above)
Comment (ShrutiLakhani) [reply Navigation.css]: Done

File: Navigation.css (hunk 3 above)
Comment (ShrutiLakhani) [reply Navigation.css]: Done.

File: Navigation.css (hunk 4 above)
Comment (ShrutiLakhani) [reply Navigation.css]: done

File: Navigation.css (hunk 5 above)
Comment (ShrutiLakhani) [reply Navigation.css:55]: I had to align it to middle of Profile, honestly, this is what worked. Can you please suggest a better way?

**Comment by ShrutiLakhani** [comment]
> Hello, just like other people said, you can use color variables in place of direct colors, and comments can be removed, btw code is really clean and readable so liked it:) Rest is good to merge.

Thanks Jyotsna. Done

**Review by ashishkatiyar2404 — APPROVED** [review]

**Review by ashishkatiyar2404 — APPROVED** [review]

File: Navigation.css (hunk 5 above)
Comment (deekshithmd) [reply Navigation.css:55]: -50% didn't work?

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**